### PR TITLE
feat: Allow Google Pay to be auto configured

### PR DESCRIPTION
**Pull request by ZengJunyong**
This is quite a similar PR with https://github.com/Rebilly/framepay-react/pull/55, actually, I almost don't know too much about React. Need to deploy Google Pay [PR](https://github.com/Rebilly/framepay/pull/507) first.

I have tested it in my local development, check this short video: https://www.loom.com/share/946c18da4d534ba781ec2329c239788d

### CODE REVIEW

File: src/lib/components/injector.tsx [hunk 1]
```diff
@@ -210,6 +212,25 @@ const elementsFabric = (type: PaymentElements): object => {
...
+                    Rebilly: makeRebillyProps(data)
+                } as GooglePayProps)
+        );
+
+        return {
+            GooglePayElement
+        };
+    }
+
     if (type === 'digitalWallet') {
```
Comment (Weetbix) [inline src/lib/components/injector.tsx:234]: Shouldn't the new google pay stuff replace all the digital wallet files, like in the framepay-vue PR?

**Review by Weetbix — CHANGES_REQUESTED** [review]

File: src/lib/components/injector.tsx (hunk 1 above)
Comment (ZengJunyong) [reply src/lib/components/injector.tsx]: Sorry for my rush, I removed all the `digital wallet` concepts.

**Review by Weetbix — CHANGES_REQUESTED** [review]
Looks good, just needs the props exported

File: src/index.ts [hunk 2]
```diff
@@ -27,10 +27,10 @@ export {
     withFramePayBankComponent,
     withFramePayIBANComponent,
     withFramePayApplePayComponent,
+    withFramePayGooglePayComponent,
     FramePayComponentProps,
     FramePayCardProps,
     FramePayBankProps,
     FramePayIBANProps,
     FramePayApplePayProps,
-    FramePayDigitalWalletProps
```
[the remark this answers is not in the document]
Comment (ZengJunyong) [reply src/index.ts:35]: Done in https://github.com/Rebilly/framepay-react/pull/56/commits/e2064ef04a5f9224f95e8d4da657600cbe8f983f

**Review by Weetbix — APPROVED** [review]

**Pull request merged**